### ISSUE TITLE
Same game in different region uses USA or Europe Box art

### ISSUE THREAD

**Issue by Panderner** [mupen64plus-ae/mupen64plus-ae#813]
Multiple games released in USA, Europe and Japan uses different Box Art but the ROM menu uses USA Box art in most games released in multiple regions. Some games like Rakugakids and Air Boarder 64 which is never released in the US uses European Box Art in Japan version of the game.

![Screenshot_2020-04-17-20-27-35-53_9784a853058524697766614cb28dab3d](https://user-images.githubusercontent.com/37503397/79569628-8927a200-80ea-11ea-904b-0968a87b0bfe.png)

**Comment by fzurita** [comment]
Yeah, I'm aware of this. The box art works best for USA version of games. This is because the server is missing a lot of the box art for other regions.

I can probably go through the effort of obtaining the  box art for other regions and make.

**Comment by Panderner** [comment]
How about the other region box arts in GameFAQs?

**Comment by Panderner** [comment]
Animal Crossing and Sin and Punishment are never released in the USA, uses Fake USA box art.

**Comment by fzurita** [comment]
Ok, I didn't know gamefaq had box art for download. I'll check it, thanks!

**Comment by spotanjo3** [comment]
Wrong box. right game. How do I replace it on my own ? Your settings changed after update. Now I am lost.

**Comment by fzurita** [comment]
Select the game, go to settings, you can change it there.

**Comment by spotanjo3** [comment]
Thanks. And done!

One thing.. which plugin should I change it ? When I press Start to playing it and I noticed the puzzles on the screen pop up seems to have gfx errors. However in game seems fine.

**Comment by fzurita** [comment]
It really depends on the game and what your device can do. If your phone can handle it, I would do GLideN64 based profiles.

**Comment by spotanjo3** [comment]
> It really depends on the game and what your device can do. If your phone can handle it, I would do GLideN64 based profiles.

Yes, mine can handle it. I just changed it and gfx is correct for puzzle pop up on the screen. Beautifully! Thanks!
